conf/telemetry/highspeed_rotorcraft.xml

```diff
@@ -250,16 +315,22 @@
       <message name="GUIDANCE_INDI_HYBRID"     period="0.002"/>
       <message name="HYBRID_GUIDANCE"          period="0.02"/>
       <message name="ESC"                      period="0.02"/>
-      <message name="STAB_ATTITUDE"            period="0.002"/>
+      <message name="STAB_ATTITUDE"            period="0.002"/>        
+      <message name="PPM"                      period="0.05"/>
```

Comment:
Some of these messages don't seem relevant for the vertical mode, like PPM.

Reply:
removed